update klaviyo source doc (#6687)
segmentio/segment-docs: src/connections/sources/catalog/cloud-apps/klaviyo/index.md

## Collections

> Collections are the groupings of data we received from your Klaviyo source. In your warehouse, each collection gets its own table, as well as a `tracks` table that aggregates all the events into a single table. 

Reviewer: ```suggestion
Collections are the groupings of data Segment receives from your Klaviyo source. In your warehouse, each collection gets its own table and a `tracks` table that aggregates all the events into a single table. 
```

To help avoid this duplication of profiles in Segment, you can check the box above, which will limit the profiles Klaviyo syncs with Segment to only the profiles originally created in Segment.

> Once you've selected the settings you would like to enable, click Update Segment Settings to complete the integration.

Reviewer: ```suggestion
Once you've selected the settings you would like to enable, click **Update Segment Settings** to complete the integration.
```
